src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -652,7 +652,7 @@ inline = choice
            , pLink
            , pImage
            , pCode
-           , pSamp
+           , pCodeWithClass [(T.pack "samp","sample"),(T.pack "var","variable")]
```

Comment:
Because you have the OverloadedStrings pragma set in this module, you can get rid of the `T.pack`.

Reply:
Thanks for the tip. Sorry, I was on vacation for two weeks so couldn't fix this.